sonar-plugin-api/src/main/java/org/sonar/api/server/authentication/UserIdentity.java

```diff
@@ -78,6 +84,24 @@ public String getEmail() {
     return email;
   }
 
+  /**
+   * Return true if groups should be synchronized for this user.
+   *
+   * @since 5.5
+   */
+  public boolean shouldSyncGroups() {
```

Comment:
I don't think that this method and field are required. Testing if `groups` is null is enough.